parser.go

```diff
@@ -1609,13 +1609,13 @@ func parseCommand(frame Frame) (node, Frame, *ParserError) {
 			".template:": parseTemplate,
 			"len":        parseLen,
 			"link:":      parseLink,
-			"unlink":     parseUnlink,
+			"unlink:":    parseUnlink,
 			"print":      parsePrint,
 			"man":        parseMan,
 			"cd":         parseCd,
 			"tree":       parseTree,
 			"ui.":        parseUi,
-			"camera.":    parseCamera,
+			"camera":     parseCamera,
```

Comment:
why ? if you want to do this you need to adapt the parseCamera function, because for now it breaks the camera command, as shown by the unit tests

Reply:
At the time this wasn't working. Now I believe it may have been fixed by the new whitespace parsing merge